### ISSUE TITLE
Remove Social media widgets

### ISSUE THREAD

**Issue by KSNINJA** [fossasia/gci18.fossasia.org#577]
**Actual Behaviour**
The social media widgets are useless on this website . They are even too small creating readability challenge . 
#### They should be removed .

**Would you like to work on the issue?**
Yes

**Comment by saarthakchats** [comment]
I support you, but I am nobody to decide 😕

**Comment by saarthakchats** [comment]
Let a mentor give his views

**Comment by Sid911** [comment]
I also agree.... I didn't want it from the first place

**Comment by AppleC4T** [comment]
I agree as well. It crams the page. Still, the mentors have the word, but if it turns out that they can't be removed, they should be at least aligned properly.
